olevba 0.60.1 on Python 3.8.10 - http://decalage.info/python/oletools
ERROR    Failed to open HEUR-Trojan.MSOffice.SAgent.gen-d2d10a2174a734249f9985a339b5b1ea0f975a4980892e9223dfe3add7a6cc42 -- probably not supported!
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/oletools/olevba.py", line 4452, in process_file
    vba_parser = VBA_Parser_CLI(filename, data=data, container=container,
  File "/usr/local/lib/python3.8/dist-packages/oletools/olevba.py", line 4011, in __init__
    super(VBA_Parser_CLI, self).__init__(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/oletools/olevba.py", line 2790, in __init__
    raise FileOpenError(msg)
oletools.olevba.FileOpenError: Failed to open file HEUR-Trojan.MSOffice.SAgent.gen-d2d10a2174a734249f9985a339b5b1ea0f975a4980892e9223dfe3add7a6cc42 is RTF, which cannot contain VBA Macros. Please use rtfobj to analyse it.
